File: C++/LCM.cpp
#include<iostream>
using namespace std;

int main(){
    int n1,n2,val;
    cout<<"Enter the numbers : ";
    cin>>n1>>n2;
    if(n1>n2){
        val=n1;
    }else{
        val=n2;
    }
    while(true){
        if((val%n1==0) && (val%n2==0)){
            break;
        }
        else{
            val++;
        }
    }
    cout<<"LCM is : "<<val;
    return 0;
}

File: Python/LCM.py
print("Enter two numbers: ",end='')
n1=int(input())
n2=int(input())
if(n1>n2):
    val=n1
else:
    val=n2
while(True):
    if((val%n1==0) and (val%n2==0)):
        break
    else:
        val=val+1
print("LCM of",n1,"and",n2,":",val)
